js/preview/src/build.ts

```diff
@@ -35,6 +35,8 @@ export async function make_build({
 			for (const [entry, path] of exports) {
 				try {
 					const x = await build({
+						root: source_dir,
```

Comment:
So weird thing but we don't actually use `root` at all because we are building in library mode, but some plugins _do_ use `root` as a reference point. Because we weren't setting it, it defaults to the current working directory which is one level higher than we need to be to find a package.json, since we run everything from the component_dir root.